app/Http/Controllers/UserAdminController.php

```diff
@@ -260,7 +260,7 @@ public function showMemberForm($id)
         $user = User::findOrFail($id);
 
         if ($user->hasSignedMembershipForm()) {
-            return Storage::download($user->member->membershipForm->generatePath(), $user->name . ' membership form.pdf');
```

Comment:
I'm not sure why this doesn't work yet, but it seems weird to stream a file we've put to CDN instead of pointing to that file in the first place. :)